labs/iot/gps-lab/client/code/raspberry-pi-gps-tracker.py

```diff
@@ -0,0 +1,95 @@
+import time
+import serial
+
+# Using the Python Device SDK for IoT Hub:
+#   https://github.com/Azure/azure-iot-sdk-python
+# The sample connects to a device-specific MQTT endpoint on your IoT Hub.
+from azure.iot.device import IoTHubDeviceClient, Message
```

Comment:
There's no instructions on installing this package. Maybe add a requirements.txt with the required packages?

Reply:
For the client side. I am encouraging students to use the default python editor Thonny, I think it will be better if we can keep it simple into a single file. I added the instructions of how to install the libraries in the MD files. However as the server code is more complex I have added a requirements.txt file.